eng/tools/RepoTasks/.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.cs]
+dotnet_diagnostic.CA1307.severity = none
+dotnet_diagnostic.CA1308.severity = none
+dotnet_diagnostic.CA1309.severity = none
```

Comment:
We should eventually fix the 3 violations in this directory and remove this, but I figured I'd at least push something that runs the "restore" step.